packages/trader/src/Stores/Modules/Trading/trade-store.ts

```diff
@@ -59,97 +191,101 @@ export default class TradeStore extends BaseStore {
     has_equals_only = false;
 
     // Underlying
-    symbol;
+    symbol = '';
     is_market_closed = false;
     previous_symbol = '';
-    active_symbols = [];
+    active_symbols: ActiveSymbols = [];
 
-    form_components = [];
+    form_components: string[] = [];
 
     // Contract Type
     contract_expiry_type = '';
     contract_start_type = '';
     contract_type = '';
-    contract_types_list = {};
-    trade_types = {};
+    contract_types_list: TContractTypesList = {};
+    trade_types: { [key: string]: string } = {};
 
     // Amount
     amount = 10;
     basis = '';
-    basis_list = [];
+    basis_list: Array<TTextValueStrings> = [];
     currency = '';
-    stake_boundary = {};
+    stake_boundary: TStakeBoundary = { VANILLALONGCALL: {}, VANILLALONGPUT: {} };
 
     // Duration
     duration = 5;
-    duration_min_max = {};
+    duration_min_max: TDurationMinMax = {};
     duration_unit = '';
-    duration_units_list = [];
-    expiry_date = '';
-    expiry_epoch = '';
-    expiry_time = '';
-    expiry_type = 'duration';
+    duration_units_list: Array<TTextValueStrings> = [];
+    expiry_date: string | null = '';
+    expiry_epoch: number | string = '';
+    expiry_time: string | null = '';
+    expiry_type: string | null = 'duration';
 
     // Barrier
     barrier_1 = '';
     barrier_2 = '';
     barrier_count = 0;
-    barriers = [];
+    main_barrier: ChartBarrierStore | null = null;
+    barriers: TBarriers = [];
+    strike_price_choices: string[] = [];
```

Comment:
@henry-deriv `strike_price_choices` need to be removed. we are using `barrier_choices` instead after Turbos PR https://github.com/binary-com/deriv-app/pull/7514/files#diff-ac84bdaae0a85981966f330c025656c3aaa66895a9376e738bb503d088594ec0 was released.

Reply:
removed